connector/connect/client/jvm/pom.xml

```diff
@@ -83,6 +83,19 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
hmm.. `spark-connect-common_2.12-3.5.0-SNAPSHOT-tests.jar ` is empty jar. why need add this  dependency?

Reply:
OK, I see the `TestUDFs`